source/common/stats/stat_merger.cc

```diff
@@ -99,9 +99,7 @@ void StatMerger::mergeGauges(const Protobuf::Map<std::string, uint64_t>& gauges,
     //    will think the gauge is NeverImport due to a code change. But for
     //    now we will leave the gauge in the child process as
     //    import_mode==Uninitialized, and accumulate the parent value in
-    //    gauge_ref.add(). Gauges in this mode will not be included in
```

Comment:
I think rather than removing this paragraph we can change "will not" to "will" as well as fix the last sentence.